### PR TITLE
Add "Open in Google Cloud Shell" button to README file.

### PR DESCRIPTION
**Pull request by noerog**

### CODE REVIEW

File: iot/mqtt-ciotc/README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 This sample app demonstrates connecting to Google Cloud IoT Core and publishing
 a single message.
 
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/cpp-docs-samples&page=editor&open_in_editor=iot/mqtt-ciotc/README.md
+
```
Comment (SurferJeffAtGoogle) [inline iot/mqtt-ciotc/README.md]: Have you confirmed that the build instructions work in cloud shell?

**Comment by noerog** [comment]
The build instructions using the Docker image work. I haven't tried installing the dependencies separately.

**Comment by tmatsuo** [comment]
@SurferJeffAtGoogle So what's the decision on this PR?

**Comment by tmatsuo** [comment]
@noerog Thanks for the PR. I tried that link, but I must say it's just more confusing to users for several reasons.

- User starts from the top level directory in the repo while the UI shows the README in `iot/mqtt-ciotc`
- I don't know what to do. Should I follow the instructions for installing the dependencies? or Should I follow the Docker instructions?
- Even after that, I don't know what to do.

I suggest that you add a special README file for "Open in Google Cloud Shell" so that the user experience will be greater.

**Comment by tmatsuo** [comment]
Closing due to inactivity

**Pull request closed without merging**